ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -487,6 +488,7 @@ public static CCDCase getCCDLegalCase() {
             .applicants(applicants)
             .respondents(respondents)
             .claimSubmissionOperationIndicators(getDefaultCCDClaimSubmissionOperationIndicators.get())
+            .state("open")
```

Comment:
Prefer to use `ClaimState.OPEN.getValue()` instead of string here

Reply:
done